update json-ld examples

tcole3 opened the pull request:
Updated json-ld examples and added text describing these examples;
Also updated abstract and introduction
@BigBlueHat if edits to abstract and introduction don't conflict with your work in progress, please merge.

tcole3 committed:
update json-ld examples

Updated json-ld examples and added text describing these examples;
Also updated abstract and introduction

tcole3 commented:
Marked as non-substantive for IPR from ash-nazg.

BigBlueHat commented:
Rawgit: https://rawgit.com/tcole3/web-annotation/77d0d1b6e068e23eedb92c595e86a17932ef6e21/serialization-html-note/index-respec.html

BigBlueHat commented on serialization-html-note/index-respec.html:
```diff
@@ -436,64 +490,83 @@
                     "type": "Person",
-                    "email": "mail-to:t-cole3@illinois.edu",
-                    "name": "Tim Cole"
+                    "email": "mail-to:mara@example.org",
```
Psuedo protocol is actually `mailto:`

BigBlueHat commented:
@tcole3 there's some good stuff in here, but some things I'd like to undo are:
 - text in the intro suggesting that one can only annotate "resources found on the Web"
   - the Model (and friends) are about annotating *anything* and a means to express those annotation *on the Web*
 - the inclusion of the Web Annotation Protocol mention is a bit distracting (the Note doesn't reference it agian)
 - I'd not put the media type in the intro...clutter-y

Otherwise, I think these are good changed.

Also, the addition of the use cases with links to places where they might be used is a great addition. Thank you!

tcole3 committed:
Update index-respec.html

- removed explicit media type+profile parameter from intro, but left reference to protocol since that is the only place the profile parameter is specified.  
- fixed mailto psuedo mime type
- add sub section for each json-ld example so would show up in TOC

BigBlueHat commented:
@tcole3 changes look good, thanks!
